### PR TITLE
Add untranslated(developer) review mode

### PR DESCRIPTION
**Pull request by ls-arnaud-levallois**
We are switching our translation tool to transifex. All languages were not translated (and are still not), and we need default values (those from the source language file). So this PR add an 'untranslated' mode

Could you also do the release of a new version, please? I can also do it if it ok with you (it only needs to run the release script from a release branch, with an increased version in package.json, right?) 

Thank you very much guys!

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -8,10 +8,11 @@ import { inspect, promisify } from 'util';
 
 const readFileAsync = promisify(readFile);
 
-type InputMode = 'translated' | 'reviewed' | 'proofread';
-type TxMode = 'translated' | 'reviewed' | 'reviewed2';
+type InputMode = 'untranslated' | 'translated' | 'reviewed' | 'proofread';
+type TxMode = 'developer' | 'translated' | 'reviewed' | 'reviewed2' ;
```
Comment (tombertrand) [inline src/utils.ts:12]: :oops: extra space..

I also feel like we should update this to match all the current available modes
>developer | translator | reviewed | onlytranslated | onlyreviewed | sourceastranslation

https://docs.transifex.com/client/pull

Comment (ls-arnaud-levallois) [reply src/utils.ts:12]: sorry for the extra space 😅 
If we give access to all of the mode, maybe can we remove the mapping, and take the transifex mode directly?

Comment (tombertrand) [reply src/utils.ts:12]: yea.. wondering why there was such restriction in the first place..? maybe for typing? @kshahani

Comment (ls-arnaud-levallois) [reply src/utils.ts:12]: Hi @tombertrand @kshahani 
any news concerning this?

Comment (kshahani) [reply src/utils.ts:12]: Yup just typing. We can continue to push this through as-is and circle back to add additional modes. ✅

**Pull request merged**